ContextDesktop/src/main/java/com/github/migi_1/Context/model/entity/IDisplayable.java

```diff
@@ -0,0 +1,39 @@
+package com.github.migi_1.Context.model.entity;
+
+import com.jme3.math.Vector3f;
+import com.jme3.scene.Spatial;
+
+/**
+ * Interface for all items that are displayed.
+ *
+ * @author Marcel
+ *
+ */
+public interface IDisplayable {
+
+    /**
+     * Get the physical model of the object.
+     * @return The physical model.
+     */
+    Spatial getModel();
+
+    /**
+     * Set the physical model of the object.
+     * @param model the physical model
+     */
+    void setModel(Spatial model);
+
+    /**
+     * Moves the entity in the given direction.
+     * @param location of where to move the entity to.
+     */
+    void move(Vector3f location);
```

Comment:
A spatial already has a move method. So forcing Displayables to implement this method is cruel. Instead what you can do is remove this method all together, or make it a default method and return getModel().move(location).